packages/next/next-server/server/incremental-cache.ts

```diff
@@ -189,7 +190,7 @@ export class IncrementalCache {
     pathname = normalizePagePath(pathname)
     this.cache.set(pathname, {
       ...data,
-      revalidateAfter: this.calculateRevalidate(pathname),
+      revalidateAfter: this.calculateRevalidate(pathname, new Date().getTime()),
```

Comment:
Continue to use the current time when calculating the revalidateAfter timestamp within `IncrementalCache.set` as this should always be considered freshly generated content.